starter-core/src/main/java/io/micronaut/starter/feature/ci/workflows/aws/templates/buildSpec.rocker.raw

```diff
@@ -0,0 +1,42 @@
+@import io.micronaut.starter.options.JdkVersion
+@import io.micronaut.starter.options.BuildTool
+
+@args (
+String projectName,
+JdkVersion jdkVersion,
+BuildTool buildTool
+)
+
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      java: corretto@jdkVersion.majorVersion()
+  pre_build:
+    commands:
+      - echo Nothing to do in the pre_build phase...
+  build:
+    commands:
+      - echo Build started on `date`
+@if (buildTool.equals(BuildTool.MAVEN)) {
+      - mvn install
```

Comment:
The equivalent of `gradle build` is `mvn verify`. `mvn install` would be comparable to `gradle publishToMavenLocal`, which for the scope of a CI job I think it's not necessary.

Also, shouldn't this be using the wrapper?

```suggestion
      - ./mvnw verify
```

Reply:
@alvarosanchez I made the change to verify you suggested. I should probably make sure it's consistent for the Github and GCP CI workflow features too.

I tried using the maven wrapper and I get errors from AWS CodeBuild. If there is a way to make that work in the `buildspec.yml` file I will make the change.

```
[INFO] 
[INFO] -------------------------< awscimvn:awscimvn >--------------------------
[INFO] Building awscimvn 0.1
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  15.813 s
[INFO] Finished at: 2022-06-23T21:43:27Z
[INFO] ------------------------------------------------------------------------
[ERROR] Unknown lifecycle phase "/root/.m2". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/LifecyclePhaseNotFoundException

[Container] 2022/06/23 21:43:27 Command did not exit successfully ./mvnw verify exit status 1
[Container] 2022/06/23 21:43:27 Phase complete: BUILD State: FAILED
[Container] 2022/06/23 21:43:27 Phase context status code: COMMAND_EXECUTION_ERROR Message: Error while executing command: ./mvnw verify. Reason: exit status 1
[Container] 2022/06/23 21:43:27 Entering phase POST_BUILD
[Container] 2022/06/23 21:43:27 Running command echo Build completed on `date`
Build completed on Thu Jun 23 21:43:28 UTC 2022
```

Reply:
The wrapper error is weird. Looks like if it were executing `mvn /root/.m2`.

If `mvn` comes preinstalled with CodeBuild, then I think it's fine to use it.

Reply:
Updated the suggestion snippet to use `mvn verify`. Apply it at your convenience.